packages/expo-image-manipulator/src/validators.ts

```diff
@@ -107,6 +116,9 @@ export function validateSaveOptions({ base64, compress, format }: SaveOptions):
     if (compress < 0 || compress > 1) {
       throw new TypeError('The "compress" argument must be a number between 0 and 1');
     }
+    if (format === SaveFormat.PNG) {
+      console.warn('The "compress" argument is not supported with png format');
```

Comment:
Seems unnecessary

Reply:
If that's the unreachable state then we can confidently remove it 💪